django_celery_results/managers.py

```diff
@@ -88,7 +88,7 @@ def get_all_expired(self, expires):
     def delete_expired(self, expires):
         """Delete all expired results."""
         with transaction.atomic():
-            self.get_all_expired(expires).delete()
+            raw_delete(queryset=self.get_all_expired(expires))
```

Comment:
another thing i want to be assured that running makemigrations detect any change in migrations

Reply:
I added [similar test for `django-silk` two days back](https://github.com/jazzband/django-silk/pull/495/files), where i simply called management command `makemigrations silk --check --dry-run` - if there are pending migrations then it returns with exit code 1.

Should we implement the same?

Reply:
that would be great!

Reply:
There is already this test which does that.

https://github.com/celery/django-celery-results/blob/0a00044f25f7de7ff73c995727b69ff25c66f25d/t/unit/test_migrations.py#L26-L32

Reply:
should we replace this with `call_command` thing i mentioned above? One advantage is that it is a higher level Django API - changes in Django internal will not affect the test - until only the command behaviour is changed.

Reply:
which part do you  want to replace with what? can you elaborate please?

Reply:
which part do you  want to replace with what? can you elaborate please?

Reply:
Running `makemigrations django_celery_results --check --dry-run` command inside the test as mentioned above.

Reply:
ok

Reply:
made those changes. Please review.

Reply:
commit - https://github.com/celery/django-celery-results/pull/235/commits/113ac2e12e7eace5619f8855e0ff69a76b8f7996